runtime/frequency/src/lib.rs

```diff
@@ -65,9 +66,14 @@ pub use common_runtime::{
 	weights,
 	weights::{BlockExecutionWeight, ExtrinsicBaseWeight},
 };
+#[cfg(feature = "frequency")]
+use frame_support::traits::Contains;
 
 /// Basefilter to only allow specified transactions call to be executed
+#[cfg(feature = "frequency")]
 pub struct BaseCallFilter;
+
+#[cfg(feature = "frequency")]
```

Comment:
Only enabled when mainnet is built

Reply:
I wish this were more clear what it applied to instead of being duplicated, but it works.

Reply:
Applied a simpler and much intuitive pattern for this now